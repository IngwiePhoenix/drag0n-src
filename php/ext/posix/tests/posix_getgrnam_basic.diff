002+ array(4) {
003+   ["name"]=>
004+   string(0) ""
005+   ["passwd"]=>
006+   string(0) ""
007+   ["members"]=>
008+   array(0) {
009+   }
010+   ["gid"]=>
011+   int(0)
012+ }
002- bool(false)
004- bool(false)
014+ array(4) {
015+   ["name"]=>
016+   string(0) ""
017+   ["passwd"]=>
018+   string(0) ""
019+   ["members"]=>
020+   array(0) {
021+   }
022+   ["gid"]=>
023+   int(0)
024+ }