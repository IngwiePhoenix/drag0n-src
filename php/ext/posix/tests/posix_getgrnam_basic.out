Basic test of POSIX posix_getgrnam function
array(4) {
  ["name"]=>
  string(0) ""
  ["passwd"]=>
  string(0) ""
  ["members"]=>
  array(0) {
  }
  ["gid"]=>
  int(0)
}
bool(false)
array(4) {
  ["name"]=>
  string(0) ""
  ["passwd"]=>
  string(0) ""
  ["members"]=>
  array(0) {
  }
  ["gid"]=>
  int(0)
}
===DONE===